cmd/buildah/main.go

```diff
@@ -19,6 +20,11 @@ func main() {
 	if buildah.InitReexec() {
 		return
 	}
+	storageOptions, err := util.GetDefaultStoreOptions()
```

Comment:
I think this won't work in buildah, as it uses the `rootless` package from libpod while buildah uses its own unshare reexec module.

Reply:
I'm at least seeing the proper content in the help.  And rootless seems to work.

```
$ ./buildah --help
...
   --root value                          storage root dir (default: "/home/dwalsh/.local/share/containers/storage")
   --runroot value                       storage state dir (default: "/run/user/3267/run")
```
Versus
```
$ buildah --help
...
   --root value                          storage root dir (default: "/var/lib/containers/storage")
   --runroot value                       storage state dir (default: "/var/run/containers/storage")
```